### PR TITLE
The building class

### PR DESCRIPTION
**Pull request by magicprog-web**

### CODE REVIEW

File: csharp/TheAccountClass/TheAccountClass/Account.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            _accountNumber = ++_accountQty;
+            _owner = owner;
+        }
+
+        public void Credit(double amount)
+        {
+            _balance += amount;
+        }
+        public void Credit(double amount, Account account)
+        {
```
Comment (paolopiaggio) [inline csharp/TheAccountClass/TheAccountClass/Account.cs]: this method can be wrapped inside a Service Class, like PaymentService
SubmitPayment(Account from, Account to, double amount);

File: csharp/TheAccountClass/TheAccountClass/Account.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            _balance += amount;
+            account._balance -= amount;
+        }
+        public void Debit(double amount)
+        {
+            _balance -= amount;
+        }
+        public void Debit(double amount, Account account)
```
Comment (paolopiaggio) [inline csharp/TheAccountClass/TheAccountClass/Account.cs]: same as above:
this method can be wrapped inside a Service Class, like PaymentService
SubmitPayment(Account from, Account to, double amount);

File: csharp/TheBuildingClass/UnitTestProject1/Building.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
```
Comment (paolopiaggio) [inline csharp/TheBuildingClass/UnitTestProject1/Building.cs]: it would be nice to rename the project UnitTestProject1

Comment (paolopiaggio) [reply csharp/TheBuildingClass/UnitTestProject1/Building.cs]: about naming conventions, generally the structure for tests is something like:
* \<projName\>
    * \<folder\>
        * \<className\>
* \<projName\>.Unit.Tests
   * \<folder\>
        * \<className\>Tests

this way it's always recognisable if the class is the test class or the actual implementation

File: csharp/TheAccountClass/TheAccountClass/Account.cs (hunk 1 above)
Comment (magicprog-web) [reply csharp/TheAccountClass/TheAccountClass/Account.cs]: Will be changed in https://github.com/nicolaspujol/Learning/pull/6

File: csharp/TheAccountClass/TheAccountClass/Account.cs (hunk 2 above)
Comment (magicprog-web) [reply csharp/TheAccountClass/TheAccountClass/Account.cs]: Will be changed in https://github.com/nicolaspujol/Learning/pull/6

**Pull request merged**